test/src/test/java/hudson/security/HudsonPrivateSecurityRealmFIPSTest.java

```diff
@@ -25,12 +25,8 @@
 package hudson.security;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.Matchers.startsWith;
+import static org.hamcrest.Matchers.*;
 import static org.junit.Assert.assertThrows;
-
```

Comment:
Don't change the format of the file, please. Revert the  deletion. This together with the wildcard makes me think you should review your IDE settings

Reply:
done